src/task/CleanStoppedSessionTask.ts

```diff
@@ -56,6 +56,9 @@ export default class CleanStoppedSessionTask extends BaseTask {
       this.context.turnCredentials = null;
       this.context.videoSubscriptions = null;
       this.context.transceiverController.reset();
+
+      // This should really be a _device selection_ operation that chooses no device,
+      // allowing the device controller to clean up any selected transform device.
```

Comment:
Should we add a TODO here if we want to defer this?

Reply:
It's not a TODO, it's an observation. We can't really fix it, but a reader will learn context and perspective by reading this comment.

Reply:
What do you mean we can't fix it? When you say "This should be" I thought it would be something we want to do (maybe in v3)

Reply:
We can't fix it within the current API because `CSST` only knows about `MediaStreamBroker`, not about `DeviceController` — it only knows how to release media streams that are tracked in the `AudioVideoControllerState`, not how to unselect a device.

The issue here is that _we now work with much more than streams_, and this API hasn't kept pace with the complexity of the SDK (something we missed in v2.0).

It's currently up to the developer's application to manage which device is currently selected (peep the complexity in the demos) and `DDC` has to figure out from the stream passed here which device to clean up.

Fixing either of those things can't be done until v3.0 at least, and it's a much larger scope than I would normally put in a TODO.